app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardActivityPresenter.kt

```diff
@@ -62,6 +62,11 @@ class RevisionCardActivityPresenter @Inject constructor(
     }
     subscribeToSubtopicTitle()
 
+    binding.actionBottomSheetOptionsMenu.setOnClickListener {
+      val bottomSheetOptionsMenu = BottomSheetOptionsMenu()
+      bottomSheetOptionsMenu.show(activity.supportFragmentManager, bottomSheetOptionsMenu.tag)
```

Comment:
This should also be ``showNow``--ditto for other showings in the app.

Reply:
Done